connectors/douban.fm.js

```diff
@@ -1,22 +1,14 @@
 'use strict';
-
-Connector.playerSelector = '.player-wrapper';
-
-Connector.trackArtSelector = '.playing-cover img';
-
-Connector.getTrack = () => {
-	return $('.player-wrapper .middle>div:nth-child(2) a').first().text();
-};
-
-Connector.getArtist = () => {
-	return $('.player-wrapper .link.artist-name').first().text();
-};
-
-Connector.getDuration = () => {
-	let text = $('.time').text();
-	return Util.stringToSeconds(text.substring(1));
-};
-
-Connector.isPlaying = () => {
-	return $('.player-wrapper .buttons .icon-pause').length > 0;
-};
+const update = (attrs) => {
+	for (let key in attrs) {
+		Connector[key] = attrs[key];
+	}
+};
+update({
+	playerSelector: '.player-wrapper',
+	trackArtSelector: '.playing-cover img',
+	trackSelector: '.middle>div:nth-child(2)>a',
+	artistSelector: '.middle>div:nth-child(3) a',
+	playButtonSelector: '.icon-play',
```

Comment:
Play/pause doesn't work for me. Have you the same issue?

Btw, I fixed it for myself by adding `isPlaying` function:
```js
Connector.isPlaying = () => $('.icon-pause').length > 0;
```

Reply:
I'm not sure what's expected.
No test report was triggered and the record was scrobbled after pausing in both cases.

Reply:
The extension should not scrobble paused tracks.

Reply:
Still the track was scrobbled, ten minutes after first detection.